rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/project/ProjectController.java

```diff
@@ -1227,4 +1227,48 @@ public void getUserProjectCount(HttpServletResponse response) throws TException
             throw new SW360Exception(e.getMessage());
         }
     }
+    @RequestMapping(value = PROJECTS_URL + "/{id}/summaryAdministration", method = RequestMethod.GET)
+    public ResponseEntity<EntityModel<Project>> getAdministration(
+            @PathVariable("id") String id) throws TException {
+        User sw360User = restControllerHelper.getSw360UserFromAuthentication();
+        Project sw360Project = projectService.getProjectForUserById(id, sw360User);
+        Map<String, String> sortedExternalURLs = CommonUtils.getSortedMap(sw360Project.getExternalUrls(), true);
+        sw360Project.setExternalUrls(sortedExternalURLs);
+        HalResource<Project> userHalResource = createHalSummaryAdministration(sw360Project, sw360User);
+        sw360Project.unsetLinkedProjects();
+        sw360Project.unsetReleaseIdToUsage();
+        return new ResponseEntity<>(userHalResource, HttpStatus.OK);
+    }
+
+    private HalResource<Project> createHalSummaryAdministration(Project sw360Project, User sw360User) throws TException {
```

Comment:
createHalSummaryAdministration and createHalProject seems to be similar methods, can you try to reuse createHalProject instead of creating a new method with similar functionality.

Reply:
changes done.